src/main/java/graphql/schema/visitor/GraphQLSchemaTraversalControl.java

```diff
@@ -0,0 +1,81 @@
+package graphql.schema.visitor;
```

Comment:
We have a different "control" that SchemaTraversalControl - so we can have better methods and "hide" the `changeNode` malarky we must do.

The only way to get a GraphQLSchemaTraversalControl (which is package level on creation) is by the helper methods and hence this simplifies the call API for consumers